ingest/ingest_pipeline.py

```diff
@@ -230,30 +226,39 @@ def load(
             return 1
         return 0
 
-    # @profile
-    def load_expression_file(self, models, is_gene_model=False):
-        collection_name = self.matrix.COLLECTION_NAME
-        # Creates operations to perform for bulk write
-        bulk_operations = list(map(lambda model: InsertOne(model), models))
+    def load_expression_file(self, gene_docs, data_array_documents):
+        self.error_logger.error(f'Starting to load expression file', extra=self.extra_log_params)
```

Comment:
Small nit - this should probably be a different log level other than `error`, but really we just want to ensure that it's logged somewhere.